static/src/javascripts/projects/common/modules/experiments/tests/no-social-count.js

```diff
@@ -0,0 +1,49 @@
+define([
+    'common/utils/config',
+    'bean'
+], function (
+    config,
+    bean
+) {
+    return function () {
+        var module = this;
+
+        this.id = 'NoSocialCount';
+        this.start = '2016-08-25';
+        this.expiry = '2016-09-21';
+        this.author = 'Gareth Trufitt';
+        this.description = 'Test whether the social share counts make a difference to the number of clicks on social buttons';
+        this.audience = 0.5;
+        this.audienceOffset = 0.5;
+        this.successMeasure = 'No significant difference in clicks between the variant and control';
+        this.audienceCriteria = 'All users';
+        this.dataLinkNames = '';
+        this.idealOutcome = 'Social share counts *do not* cause a significant increase in sharing, so we can remove them';
```

Comment:
the ideal outcome in statistical sense should be total clicks on all share buttons in the top article meta decreases by less than 2%?